dm/master/election.go

```diff
@@ -63,23 +55,9 @@ func (s *Server) electionNotify(ctx context.Context) {
 
 				ok := s.startLeaderComponent(ctx)
 
-				failpoint.Inject("FailToStartLeader", func(val failpoint.Value) {
-					masterStrings := val.(string)
-					if strings.Contains(masterStrings, s.cfg.Name) {
-						log.L().Info("fail to start leader", zap.String("failpoint", "FailToStartLeader"))
-						ok = false
```

Comment:
Just curious 😕 Why failpoint can't change ok to false here?

Reply:
en.... After investigation, failpoint generate
```
if _, ok := failpoint.Eval(_curpkg_("failpointName")); ok {
		ok=false
	}
```
and the `ok` override the outer ok. 😂
so actually we can just rename the variable....

Reply:
emmmm 🤔 get it

Reply:
the name of `ok` is not OK. :scream: